modules/node_modules/@ncigdc/components/DownloadButton.js

```diff
@@ -28,14 +31,20 @@ const DownloadButton = ({
   filters = {},
   active,
   setActive,
+  altMessage = false,
+  style = {},
   extraParams = {},
+  dispatch,
 }) => {
-  const icon = <i key="icon" className={`fa fa-download ${active ? 'fa-spinner fa-pulse' : ''}`} />;
+  const icon = active ? <Spinner /> : <DownloadIcon />;
```

Comment:
when ever components are in an array in react they need a unique `key` prop so it knows when they change. (it's put in an array here https://github.com/NCI-GDC/portal-ui/blob/6a5d2b1d1e40b4ecc4d11b816a2e7c7d4fe04a2c/modules/node_modules/%40ncigdc/components/DownloadButton.js#L77 )